test-graphics/TestGraphics.cpp

```diff
@@ -40,13 +40,13 @@ NAS2D::State* TestGraphics::update()
 
 	{
 		const auto jitter = [&]() -> decltype(auto) {
-			std::uniform_int_distribution<int> dist(0, 64);
+			std::uniform_int_distribution<uint8_t> dist(0, 64);
 			return dist(generator);
 		};
-		for (int i = 0; i < 2000; ++i)
+		for (auto i = 0u; i < 2000u; ++i)
```

Comment:
Uhh, was this really necessary? The loop variable isn't even used anywhere.

Though to be fair, I do tend to prefer unsigned when possible.

Reply:
The test is kinda bad in the first place. I don't even know what it's supposed to be testing.